### PR TITLE
[GR-1471] Call Ready TAR uses Mean Bait Coverage

### PR DESCRIPTION
**Pull request by slazicoicr**
To make Dashi graphs match QM-024. Wiki has been updated.

Jira ticket or GitHub issue:

- [X] Updates CHANGELOG.md
- [X] Updates dev docs if applicable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and as of version 1.0.0, follows semantic versioning.
 
 ## [Unreleased]
+  * Switched Call Ready TAR Median Target Coverage to Mean Bait Coverage
+  * No data for swap view doesn't cause crash (stage cache has become empty)
```
Comment (avarsava) [inline CHANGELOG.md:10]: These changes would optimally be in different commits so that the git blame for the relevant lines aren't confusing down the line

**Review by avarsava — APPROVED** [review]

**Review by callunity — APPROVED** [review]
